<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceWidget</class>
 <widget class="QWidget" name="SourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>63</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>936</width>
    <height>63</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SourceWidget</string>
  </property>
  <widget class="QLabel" name="lblSWFPS">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSW">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbSW">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string> %v|%m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSWChoose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Movie</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
